README.md

```diff
@@ -191,8 +191,63 @@ Packages found only in gcr.io/google-appengine/python:2017-06-29-190410: None
 Version differences: None
 
 ```
+## Example Run with json post-processing
```

Comment:
@nkubala @aaron-prindle any preferences on including vs not including the python script? I had gotten the vibe that an example would include the actual post-processing script as a full example...